itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/txn/compactor/CompactorOnTezTest.java

```diff
@@ -95,6 +98,10 @@ private void setupTez(HiveConf conf) {
     conf.set("hive.tez.container.size", "128");
     conf.setBoolean("hive.merge.tezfiles", false);
     conf.setBoolean("hive.in.tez.test", true);
+    if (!mmCompaction) {
+      conf.set("tez.grouping.max-size", "1024");
```

Comment:
Can we leave a comment why are these settings needed?

Reply:
Sure, added a comment.